src/main.rs

```diff
@@ -90,7 +90,9 @@ impl Shop {
 
 /// A Card is a card stores a price, health, and damage.
 struct Card {
-    // TODO: Add fields to this struct
+    price: i128,
+    health: i128,
+    damage: i128,
```

Comment:
I'd probably recommend using `i32` here, since it's a pretty good default :) I used `i128` last time to show off how working with real big numbers works, but we don't need to use it in most cases :+1: